src/main/java/be/syntra/devshop/DevshopFront/services/ProductService.java

```diff
@@ -0,0 +1,22 @@
+package be.syntra.devshop.DevshopFront.services;
+
+import be.syntra.devshop.DevshopFront.models.SaveStatus;
+import be.syntra.devshop.DevshopFront.models.dto.ProductDto;
+
+public interface ProductService {
+
+    /***
+     * creates a Dto of a Product obj
+     * to use in the form when creating a new Product
+     * @return
```

Comment:
fill in what the method will return (you did so in the two lines above). Or remove the annotation